src/azure-cli/azure/cli/command_modules/iot/_help.py

```diff
@@ -402,6 +402,9 @@
   - name: Create an IoT Hub with the standard pricing tier S1 and 4 partitions, in the 'westus' region, with tags.
     text: >
         az iot hub create --resource-group MyResourceGroup --name MyIotHub --location westus --tags a=b c=d
+  - name: Create an IoT Hub with a system-assigned managed identity, and assign a role and scope to a storage account for the created identity.
+    text: >
+        az iot hub create --resource-group MyResourceGroup --name MyIotHub --location westus --assign-identity [system] --role "Storage Blob Data Contributor" --scopes {resourceId}
```

Comment:
Why the value sample of parameter `--assign-identity` is `[system]`, what does `[]` mean?

Reply:
`[system]` is a string used to refer to the system-assigned managed identity, we wanted to keep in line with other services using the same parameter - just a few other examples:
https://github.com/Azure/azure-cli/blob/f30478cd70216b299c1bc250777ae269c3f8198b/src/azure-cli/azure/cli/command_modules/appconfig/custom.py#L28

![image](https://user-images.githubusercontent.com/13545962/118764366-3c068a00-b82e-11eb-92af-1eee2b002478.png)